tensorflow/core/kernels/cudnn_rnn_ops.cc

```diff
@@ -502,20 +502,23 @@ struct CudnnRnnModelShapes {
   int dir_count;
   int max_seq_length;
   int batch_size;
+  int c_num_units;
   TensorShape input_shape;
   TensorShape output_shape;
   TensorShape hidden_state_shape;
+  TensorShape c_state_shape;
   // At present only fields related to cached RnnDescriptor are concerned.
   bool IsCompatibleWith(const CudnnRnnModelShapes& rhs) const {
     return num_layers == rhs.num_layers && input_size == rhs.input_size &&
-           num_units == rhs.num_units && dir_count == rhs.dir_count;
+           num_units == rhs.num_units && dir_count == rhs.dir_count &&
```

Comment:
(Sorry I'm not familiar with the details of cudnnrnn) What is dir_count? Will this new check about dir_count break any existing model?